README.md

```diff
@@ -66,6 +75,58 @@ $ make && bin/zotonic runtests mod_ginger_collection
 
 For more, see the [Docker](docs/docker.md) doc chapter.
 
+### 2. Selective Docker (recommended)
+
+Due to limitations in Docker for Mac, file synchronization performance suffers
+when you have large amounts of files (i.e. many sites with node_modules/ 
+directories).
+
+By running Zotonic directly on your host (outside Docker) we circumvent this
+limitation. Another advantage is that you can directly make changes in Zotonic
+source code, too. 
+
+All other services (PostgreSQL, Elasticsearch, Kibana) still run in
+containers.
+
+First, install Install Erlang locally:
+
+```bash
+$ brew install erlang@20
+$ brew link erlang@20 --force
+```
+
+Then clone both Ginger and Zotonic:
+
+```bash
+$ git clone https://github.com/driebit/ginger.git
+$ git clone https://github.com/zotonic/zotonic.git --branch 0.x 
+```
+
+Copy Ginger’s configuration file, which includes its dependencies:
+
+```bash
+cp ginger/config/zotonic.config ~/.zotonic/0/zotonic.config
+```
+
+Point Zotonic to your Ginger sites/ and modules/ directories:
+
+```bash
+$ mkdir zotonic/user
+$ ln -s ../../ginger/sites zotonic/user/sites
+$ ln -s ../../ginger/modules zotonic/user/modules
+```
+
+And run Zotonic:
+
+```bash
+$ cd ginger
+$ make up-support
+$ cd ../zotonic
+$ bin/zotonic debug
+```
+
+Zotonic then runs on port 8000: http://localhost:8000.
```

Comment:
Skel uses to site.docker.test https://github.com/driebit/ginger/blob/master/skel/config.in#L3

Add note on how to solve this? 
```echo "rdr pass inet proto tcp from any to any port 80 -> 127.0.0.1 port 8000" | sudo pfctl -ef -```

Reply:
Resolved with a `make start` target that both runs `pfctl` and starts/stops Docker containers.